Components/Core/Inc/Queue.hpp

```diff
@@ -34,6 +34,8 @@ class Queue{
 
 	//Functions
 	bool Send(Command& command);
+	bool SendFromISR(Command& command);
+
```

Comment:
Make a function to automatically make a command inside the Queue, would simplify code for trivial commands.

Reply:
Will be implemented later.